R/install_air.R

```diff
@@ -0,0 +1,69 @@
+#' Air Install
+#'
+#' @description checks for air installation status and installs it if
+#' required, updating the global settings if selected
+#'
+#' @importFrom data.table like
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' install_air()
+#' }
+
+install_air <- function() {
```

Comment:
I think we need a flag on the input that allows the user to prevent the code changing their global settings. I guess something along the lines of: 
`update_global_settings = TRUE`

Then obviously if that's false some of the code below get's skipped (preferably with some messaging to say we recommend updating your global settings to run Air by default).